src/hooks/useMapToReactNumberConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { useAppSelector } from 'src/hooks/useAppSelector';
+import { appLanguageStateSelector } from 'src/selectors/appLanguageStateSelector';
+import { formatNumber } from 'src/utils/formattingUtils';
+import type { IInputFormatting } from 'src/layout/layout';
+import type { CurrencyFormattingOptions, UnitFormattingOptions } from 'src/utils/formattingUtils';
+
+export const useMapToReactNumberConfig = (value: string | undefined, formatting: IInputFormatting) => {
+  const appLanguage = useAppSelector(appLanguageStateSelector);
+
+  if ((!formatting?.currency && !formatting?.unit) || !value) {
+    return formatting;
+  }
+
+  const createOptions = (config: IInputFormatting) => {
+    if (config.currency) {
+      return { style: 'currency', currency: config.currency.valuta } as CurrencyFormattingOptions;
+    }
+    if (config.unit) {
+      return { style: 'unit', unit: config.unit.unitType } as UnitFormattingOptions;
+    }
+    return undefined;
+  };
+
+  const position: string | undefined = formatting?.currency?.position || formatting?.unit?.position || undefined;
```

Comment:
should we write a small comment on what position is?🤔 😄

Reply:
I agree :)